domains/runtime/core-eth-relay/src/runtime.rs

```diff
@@ -312,6 +369,11 @@ construct_runtime!(
         Messenger: pallet_messenger = 6,
         Transporter: pallet_transporter = 7,
```

Comment:
@vedhavyas looks like we'll want this indices to be constants, otherwise feels fragile. Maybe even some integration test if we have infra for that.

Reply:
I dont like it as well. We were already bitten by it. I have a plan to have a safer approach soon that should remove this dependency.